frontend/components/TableContainer/DataTable/_styles.scss

```diff
@@ -224,6 +223,42 @@ $shadow-transition-width: 10px;
             white-space: normal;
           }
         }
+        .text-cell {
+          display: inline-flex;
+        }
+        .link-cell {
+          display: block;
+          padding: $pad-small 0; // larger clickable area
+
+          &:hover {
+            text-decoration: underline;
+          }
+          > div {
```

Comment:
Is the relationship `.link-cell > div` implied by the nested structure here, making the `>` redundant?

Reply:
oops, deleting duplicate comment — PR reviews from VSCode are a little buggy!